Daytime-Nighttime-Interactions/fault-reporting.rst

```diff
@@ -24,118 +24,105 @@
 Fault Reporting
 ###############
 
-.. note::
-    This is a procedure template file that is associated with a template directory structure. 
-    This note should be deleted when the section is properly populated.
+Reporting telescope and observatory faults - whether they are mechanical errors, software bugs, or facilities issues - is a crucial aspect of observatory operations. 
+Understanding the observatory and its efficiency inside and out begins with robust fault reporting, documenting recovery, and knowledge-sharing. 
+This section describes the process to file a fault report for any incident that happens during nighttime operations in the Observing Operations `(OBS) <https://jira.lsstcorp.org/projects/OBS/issues/OBS-4?filter=allopenissues>`__ JIRA project.
 
-.. _Title-of-Procedure-Overview:
 
-Overview
-^^^^^^^^
+.. _fault-reporting-Guidelines-for-productive-reporting:
 
-.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+Guidelines for productive reporting
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-This is a template for a procedure. It is performed by authorized and trained users.
+It is important for the person filing a fault report to remember that the most important thing is that the team can understand the problem well. 
+Some guidelines to keep in mind are:
 
-.. _Title-of-Procedure-Prerequisites:
+- Facts first. The author of the fault report should provide as many details as possible, including screenshots, telescope telemetry, and timestamps for future investigation.
+- Ideas are welcome, but let the facts speak first.
+- Report a problem, not a person. Identifying the problem and reporting effecitvely ensure that the Rubin team will move forward with a solution. Identifying a person as being "at-fault" for a 
```

Comment:
New sentences should be on a new line. To do this in a bulleted list (and maintain proper formatting) you'll need to start it at the same indent as "Report".
See https://sembr.org/ if you're not familiar with semantic line breaks. They're very useful!